sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -22,22 +22,16 @@ import {
   TrackedMessageIdsInfo
 } from "./utils";
 import * as appInsights from "applicationinsights";
-const writeFile = util.promisify(fs.writeFile);
 
-appInsights.setup()
+import * as dotenv from "dotenv";
+dotenv.config();
+
+appInsights
+  .setup()
   .setUseDiskRetryCaching(true)
   .start();
-    
-const defaultClient = appInsights.defaultClient;
-
-async function appendFile(reportFileName: string, text: string) {
-  defaultClient.trackTrace({
-    message: text,
-    properties: {
-      type: "stresstest"
-    }
-  });
-}
+
+export const defaultClient = appInsights.defaultClient;
```

Comment:
Just thinking..
Should we add a switch to enable app-insights or file storage?

Reply:
Let's continue this discussion after this PR. There is (at least in .net) a way to make a custom handler in the AppInsights pipeline so we could just write one of those.